cloud/disk_manager/internal/pkg/facade/operation_service.go

```diff
@@ -34,8 +37,36 @@ func getOperation(
 
 	switch result := o.Result.(type) {
 	case *operation.Operation_Error:
+		status := grpc_status.FromProto(result.Error)
+		dmStatus := grpc_status.New(status.Code(), status.Message())
+
+		// We should return disk_manager.ErrorDetails, comply with contract with
+		// compute.
```

Comment:
// Convert to disk_manager.ErrorDetails via marshaling/unmarshaling in order to be compliant with disk_manager/api